src/goPath.ts

```diff
@@ -41,6 +41,15 @@ export function getBinPathWithPreferredGopath(toolName: string, preferredGopaths
 	}
 
 	const binname = alternateTool && !path.isAbsolute(alternateTool) ? alternateTool : toolName;
+	const env = Object.assign({}, process.env);
+	if (env['GOBIN'] != undefined){
```

Comment:
This `if` check can be removed. If no GOBIN is set, then `getBinPathFromEnvVar()` will return `null` anyway.